src/foam/lib/json/AnyParser.java

```diff
@@ -18,6 +18,6 @@ public AnyParser() {
         new BooleanParser(),
         new LongParser(),
         new DoubleParser(),
-        new FObjectParser()));
+        new ExprParser()));
```

Comment:
Then, add a comment to explain that here or else add each of those Alt choices here directly.